mmv1/third_party/terraform/provider/provider_mmv1_resources.go.erb

```diff
@@ -15,9 +15,6 @@ import (
 	"github.com/hashicorp/terraform-provider-google/google/services/dataflow"
 	"github.com/hashicorp/terraform-provider-google/google/services/servicenetworking"
 	"github.com/hashicorp/terraform-provider-google/google/tpgiamresource"
-	<% if version == 'ga' -%> // https://github.com/hashicorp/terraform-provider-google/issues/15633 for details
```

Comment:
With the other PRs merged, do we still need to remove this, or do we possibly need a rebase?